common/presenters/move-to-card-component.js

```diff
@@ -49,6 +49,7 @@ function moveToCardComponent({
       ? { text: i18n.t(`statuses::${status}`) }
       : undefined
     const personCardComponent = profileToCardComponent({
+      locationType: toLocation?.location_type,
```

Comment:
I don't think this will cater for both incoming and outgoing contexts. I think the location type we're actually concerned with is the location type of the currently selected location, whether than be from the session or the ID in the URL.

I think you may have to pass the `locationType` in to this via the presenters to get the correct context. I know this is making things a little tricky but I don't think the value we care about is related to the move, it comes from a higher level. Does that make sense?

Reply:
If you look at how we get the [from location type in the creation journey](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/153c6f50b69528575f790e2829c7ccbf9b7897f4/app/move/app/new/controllers/base.js#L162) we use the session value and I think this is what we should be sending to the presenter.